scripts/linux.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Install script to configure the agent on Linux nodes
+# This script is intended to be used with the Puppet Enterprise Getting Started VM
+set -e
+
+echo "Installing Puppet agent..."
+if [ -x "/opt/puppetlabs/bin/puppet" ]; then
+  echo "WARNING: Puppet agent is already installed. Re-install, re-configuration, or upgrade not supported and might fail."
+fi
+
```

Comment:
we probably need to set the hostname here

Reply:
then we will be expecting the hostname from user?

Reply:
We can set the hostname to the resource name by default.

Reply:
done!!